### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by FaganSC**
Updated limitation note for Ord-Wide Teams to note this includes Archived Teams.

### CODE REVIEW

**Comment by FaganSC** [comment]
This is an important limit to include as it's not clear that Archived Teams are included in the Org-Wide Limit of the number of teams.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ This article describes some of the limits, specifications, and other requirement
 |Number of teams a user can be a member of|1,000&sup2;|
 |Number of members in a team | 10,000<sup>5</sup>     |
 |Number of owners per team | 100   |
-|Number of org-wide teams allowed in a tenant | 5     |
+|Number of org-wide teams allowed in a tenant | 5  (includes archived Teams)   |
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:36]:
```suggestion
|Number of org-wide teams allowed in a tenant | 5  (includes archived teams)   |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/limits-specifications-teams.md (hunk 1 above)
Comment (SerdarSoysal) [reply Teams/limits-specifications-teams.md:36]: closing this suggestion. we should use a footnote we already have for this.

**Pull request merged**